components/homepage/HomepageNav.js

```diff
@@ -215,8 +236,8 @@ const HomepageNav = ({
       ref={dropdownMobileRef}
     >
       {userData !== null && userData !== undefined ? (
-        <div className="nav__mobile-profile tw:z-10">
-          <div className="nav__mobile-img">
+        <div style={{border:"1px solid red"}} className="nav__mobile-profile tw:z-10">
+          <div style={{border:"1px solid red"}} className="nav__mobile-img">
```

Comment:
since you're just adding a border and not manipulating the DOM, why not add it to the className instead, or better still use tailwind, so we can stick to classNames. for consistency. WDYT ??

Reply:
solved

Reply:
these are my oriantation borders, before I was interducing tailwind...I forgot to delete them

Reply:
Cool